### ISSUE TITLE
Can't override default_host = "" of Terraform

### ISSUE THREAD

**Issue by junichi-hata** [fastly/terraform-provider-fastly#75]
Hello,

### Terraform Version
Terraform v0.11.7
+ provider.fastly v0.1.4

### Affected Resource(s)
fastly_service_v1

### Terraform Configuration Files
```hcl
resource "fastly_service_v1" "myservice" {
  name = "xxxxxxxxxxx.global.ssl.fastly.net"

  domain {
    name = "xxxxxxxxxxx.global.ssl.fastly.net"
  }

  backend {
    address = "10.0.0.1"
    name    = "Temp"
    port    = 80
  }

  default_ttl   = 10
  force_destroy = true
  default_host = ""

  vcl {
    name    = "main"
    main    = "true"
    content = "${file("main.vcl")}"
  }
}
```

### Expected Behavior
`Override Host` header of Fastly should be deleted

### Actual Behavior
`No changes. Infrastructure is up-to-date.` by `terraform apply`

### Steps to Reproduce
0.  set `default_host = ""` in `override.tf`
1. `terraform apply`

### Important Factoids
When I set `default_host = false`, it will set `0` for `Override Host` of Fastly UI, but it deletes in actual vcl line of `set req.http.host = xxxx`

**Comment by Integralist** [comment]
Thank you for taking the time to open this issue.

I appreciate this was reported a long time ago (and I appreciate your patience awaiting our feedback) but I believe we may now have a fix in place to resolve the issue you're describing. We merged it this morning as the new minor version `v0.21.3`, here are some details about the changes:

- [Migrate Terraform provider to use latest Go-Fastly 2.0.0 release](https://github.com/fastly/terraform-provider-fastly/releases/tag/v0.21.3).
- [Key changes in Go-Fastly 2.0.0](https://github.com/fastly/terraform-provider-fastly/issues/85) that I think relate to this issue.

If possible, could you try upgrading to the latest version (`v0.21.3`) and let me know if this indeed resolves the issue.

Thanks again.

**Comment by smaeda-ks** [comment]
This has been fixed in v0.33.0:
https://github.com/fastly/terraform-provider-fastly/blob/main/CHANGELOG.md

Sorry for taking such a long time! We appreciate your feedback.
